mineinabyss-components/src/main/kotlin/com/mineinabyss/components/PlayerData.kt

```diff
@@ -14,6 +14,9 @@ class PlayerData(
     var curseAccrued: Double = 0.0,
     var exp: Double = 0.0,
     var keepInvStatus: Boolean = true,
+    var showPvPMessage: Boolean = true,
```

Comment:
Probably better to use an enum with options like `UNDECIDED`, `ENABLED`, `DISABLED`, then send the message when it is undecided and update it to disabled if not chosen in time.

Also personally I feel like it might be better to always show the message unless explicitly disabled. Thoughts on that? If so you would probably want to keep it as two booleans, one for prompt and another for enabled.

Reply:
Like have the decide yes or no, but still show it next time around?
and make it togglable via a command ingame or something so that they then take last choice or from /pvp on/off?

Reply:
Wanna use Guiy for this instead. But I will sort that at a later date. For now this should be fine. Don't really like chat clicks all too much so